elasticsearch/kibana/init.sls

```diff
@@ -1,145 +1,8 @@
-{% from "elasticsearch/kibana/map.jinja" import kibana with context %}
 {% from "elasticsearch/map.jinja" import elasticsearch with context %}
-{% set os_family = grains['os_family'] %}
-
-install_kibana_dependencies:
-  pkg.installed:
-    - pkgs: {{ kibana.pkgs }}
-    - require_in:
-        {% if elasticsearch.use_elastic_stack %}
-        - pkgrepo: configure_elastic_stack_package_repo
-        {% else %}
-        - pkgrepo: configure_kibana_package_repo
-        {% endif %}
-    - reload_modules: True
-    - update: True
 
+{% if elasticsearch.use_elastic_stack %}
```

Comment:
This shouldn't be conditional on whether the elastic stack is being used. Kibana is useful and usable against earlier versions of elasticsearch and we may not want to rely on the elastic stack repos to install Kibana.

I may be wrong in thinking this, so feel free to push back.